spec/support/vcr.rb

```diff
@@ -6,12 +6,16 @@
   config.cassette_library_dir = 'spec/fixtures/slack'
   config.hook_into :webmock
   config.configure_rspec_metadata!
+  config.filter_sensitive_data('<SLACK_API_TOKEN>') { ENV['SLACK_API_TOKEN'] }
```

Comment:
This will catch tokens anywhere - body, header, etc. - rather than explicitly filtering in the `before_record` hook

Reply:
I was looking for how to do this!